src/Stan/Cli.hs

```diff
@@ -115,6 +119,19 @@ cabalFilePathP = many $ strOption $ mconcat
     , help "Relative path to the .cabal file (can specify many of this option)"
     ]
 
+configFileP :: Parser (Maybe FilePath)
+configFileP = optional $ strOption $ mconcat
+    [ long "config-file"
+    , metavar "FILE_PATH"
+    , help "Relative path to the .toml configurations file"
+    ]
+
+useDefaultConfigFileP :: Parser Bool
+useDefaultConfigFileP = flag True False $ mconcat
```

Comment:
`optparse-applicative` should have something like [switch](https://hackage.haskell.org/package/optparse-applicative-0.15.1.0/docs/Options-Applicative.html#v:switch) but with the reverse order of flags, like, `unswitch` or `disable` or something like that :unamused: